lib/ui/bottombar/test_bottombar_navigation.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:farmsmart_flutter/data/bloc/article/ArticleListProvider.dart';
+import 'package:farmsmart_flutter/data/repositories/FlameLink.dart';
+import 'package:farmsmart_flutter/data/repositories/article/ArticleRepositoryInterface.dart';
+import 'package:farmsmart_flutter/data/repositories/article/implementation/ArticlesRepositoryFlamelink.dart';
+import 'package:farmsmart_flutter/ui/bottombar/tab_navigator.dart';
+import 'package:farmsmart_flutter/ui/discover/ArticleList.dart';
+import 'package:farmsmart_flutter/ui/playground/data/playground_datasource_impl.dart';
+import 'package:farmsmart_flutter/ui/playground/playground_view.dart';
+import 'package:farmsmart_flutter/utils/assets.dart';
+import 'package:flutter/material.dart';
+
+import 'bottom_navigation_bar_controller.dart';
+
+final cms =
+    FlameLink(store: Firestore.instance, environment: Environment.development);
+final articleRepo = ArticlesRepositoryFlameLink(cms);
+
+void main() => runApp(SampleApp());
+
+//TODO Just for testing purposes, should be removed before merge.
+class SampleApp extends StatelessWidget {
+
+  static final double bottomBarIconSize = 25;
+
+  static final Widget _discoverPage = ArticleList(
+      viewModelProvider: ArticleListProvider(
+          title: 'Discover',
+          repository: articleRepo,
+          group: ArticleCollectionGroup.discovery));
+
+  static final Widget _myPlotPage = FirstPage();
```

Comment:
drop the specific app names, make it just tab1, 2 etc

Reply:
Same as here https://github.com/farmsmart/farmsmart-flutter/pull/48#discussion_r303400891